documentation/jetty/modules/code/examples/src/main/java/org/eclipse/jetty/docs/programming/migration/ServletToHandlerDocs.java

```diff
@@ -60,10 +61,10 @@ public boolean handle(Request request, Response response, Callback callback) thr
             //   - servletRequest.getProtocol();
             String protocol = request.getConnectionMetaData().getProtocol();
 
-            // Gets the full request URI.
+            // Gets the request URL.
             // Replaces:
             //   - servletRequest.getRequestURL();
-            String fullRequestURI = request.getHttpURI().asString();
+            StringBuffer requestURL = new StringBuffer(HttpURI.build(request.getHttpURI()).query(null).asString());
```

Comment:
I think splitting makes it more clear what is the Jetty API, and what is boilerplate:

```suggestion
            HttpURI uri = HttpURI.build(request.getHttpURI()).query(null);
            StringBuffer requestURL = new StringBuffer(uri.asString());
```